src/QandaAtom.tsx

```diff
@@ -1,7 +1,445 @@
+    margin-right: 16px;
+    margin-bottom: 6px;
+    width: 100px !important;
+    height: 100px !important;
+    object-fit: cover;
+    border-radius: 50%;
+    display: block;
+    border: 0px;
+`;
+
+const iconSpacing = css`
+    display: inline-flex;
+    align-items: center;
+    ${textSans.xsmall()};
+`;
+
+const thumbIcon = css`
+    width: 16px;
+    height: 16px;
+`;
+
+const dislikeThumb = css`
+    transform: rotate(180deg);
+`;
+
+const footerStyling = css`
+    font-size: 13px;
+    line-height: 16px;
+    display: flex;
+    justify-content: flex-end;
+`;
+
+const footerSnippet = css`
+    display: flex;
+    align-items: center;
+    ${textSans.xsmall()};
+`;
+
+const footerFeedback = css`
+    ${textSans.xsmall()};
+    height: 28px;
+`;
+
+const creditStyling = css`
+    ${textSans.xsmall()};
+    margin: 12px 0;
+`;
+
+const creditIconStyling = css`
+    height: 12px;
+    overflow: visible;
+    width: 6px;
+    fill: white;
+`;
+
+const creditSpanStyling = css`
+    display: inline-flex;
+    background: #bdbdbd;
+    border-radius: 100%;
+    width: 16px;
+    height: 16px;
+    align-items: center;
+    justify-content: center;
+    margin-right: 5px;
+`;
+
+// Components
+const Figure = ({
+    id,
+    title,
+    children,
+    expandHandler,
+}: {
+    id: string;
+    title: string;
+    children: React.ReactNode;
+    expandHandler: () => void;
+}) => {
+    return (
+        <figure
+            className={figureStyling}
+            data-atom-id={id}
+            data-atom-type="qanda"
+        >
+            <details
+                data-snippet-id={id}
+                data-snippet-type="qanda"
+                className={detailStyling}
+            >
+                <Summary title={title} expandHandler={expandHandler}></Summary>
+                {children}
+            </details>
+        </figure>
+    );
+};
+
+const Summary = ({
+    title,
+    expandHandler,
+}: {
+    title: string;
+    expandHandler: () => void;
+}) => {
+    const [hasBeenExpanded, setExpandStatus] = useState(false);
+    return (
+        <summary className={summaryStyling}>
+            <span
+                className={css`
+                    color: #e00000;
+                    display: block;
+                    ${body.medium({
+                        lineHeight: 'tight',
+                        fontWeight: 'bold',
+                    })};
+                `}
+            >
+                Q&amp;A
+            </span>
+            <h4
+                className={css`
+                    ${headline.xxxsmall({
+                        fontWeight: 'medium',
+                    })};
+                    margin: 0;
+                    line-height: 22px;
+                `}
+            >
+                {title}
+            </h4>
+            <span
+                className={showHideStyling}
+                onClick={() => {
+                    if (!hasBeenExpanded) {
+                        expandHandler();
```

Comment:
I'd check that this handler isn't send multiple times - you may need to put it inside `useEffect` with the `hasBeenExpanded` state as the conditional source prop https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect (maybe not as in onClick but this has been be a source of bugs in my code before!)